tensorflow/lite/micro/examples/hello_world/README.md

```diff
@@ -40,10 +40,10 @@ bazel run :evaluate_test
 
 ```bash
 make -f tensorflow/lite/micro/tools/make/Makefile third_party_downloads
```

Comment:
we should also list the bazel command for completeness since that is where we would ideally like folks to start.

Reply:
done